app/views/shipit/commits/_commit.html.erb

```diff
@@ -5,13 +5,14 @@
     <span class="commit-title"><%= render_commit_message_with_link commit %></span>
     <p class="commit-meta">
       <span class="sha"><%= render_commit_id_link(commit) %></span>
-      <%= timeago_tag(commit.committed_at, force: true) %>
-      <span class="utc-timecode">at <%= commit.committed_at.strftime('%H:%M:%S') %> UTC</span>
       <% if commit.additions? && commit.deletions? %>
         <span class="code-additions">+<%= commit.additions %></span>
         <span class="code-deletions">-<%= commit.deletions %></span>
       <% end %>
     </p>
+    <p class="commit-meta">
+      <%= timeago_tag(commit.committed_at, force: true) %>
```

Comment:
It was there before, see: https://github.com/jgraichen/rails-timeago/blob/542cdfb91b1a866a86457a8a44b92595021c865c/lib/rails-timeago/helper.rb#L24-L30